NRzure/README.md

```diff
@@ -0,0 +1,51 @@
+[![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
```

Comment:
I think we can remove this file, overall we don't like to put extra files into this repo besides the core files needed to create a quickstart, since things can move around in here and only the required files are published. This one appears to just have information related to the creation of quickstarts themselves and does not appear to be directly related to this quickstart.